### ISSUE TITLE
Close tab page is not working

### ISSUE THREAD

**Issue by NejcZdovc** [brave/browser-laptop#9420]
- Did you search for similar issues before submitting this one?
yes
- Describe the issue you encountered:
Close tab page is not working
- Platform (Win7, 8, 10? macOS? Linux distro?):
macOS
- Brave Version (revision SHA):
7ce06eea414eab8c2eaf399129a77c9a2ac89f59
- Steps to reproduce:
    1. Set tab page limit to 6
    2. Open 10 tabs
    3. Try to close active tab page
    
- Actual result:
Nothing happens
- Expected result:
Tabs from current tab page should be closed
- Will the steps above reproduce in a fresh profile? If not what other info can be added?
yes
- Is this an issue in the currently released version?
yes
- Can this issue be consistently reproduced?
yes

**Comment by NejcZdovc** [comment]
This is not working because [this](https://github.com/brave/browser-laptop/blob/master/app/renderer/reducers/frameReducer.js#L171) if return false for new tab, blank tab and private tab. We need to change this back, so you can close tabs tab page without any restrictions.

This was introduced in commit https://github.com/brave/browser-laptop/commit/f4283c0053569842773d5d6aca91825b55732ba3#diff-e12cc9b3579286a186d0f0c82dfa8186R66

cc @bridiver

**Comment by luixxiul** [comment]
Updated the manual test plan: https://github.com/brave/browser-laptop/wiki/Manual-Tests/_compare/3eee18b5675c6a06aa5bd614d6b6f9d1ace7768e...13895aa7fab6ae3af8d7c84a7a6565b5b4fec2d1

CC @LaurenWags @alexwykoff @srirambv

**Comment by bsclifton** [comment]
This was working for me when I tested https://github.com/brave/browser-laptop/pull/9414.  Is this still an issue, @NejcZdovc?

**Comment by NejcZdovc** [comment]
Yes it's still an issue, as reported in #9609

**Comment by luixxiul** [comment]
If the tab page has two tabs, `about:preferences` and the new tab, clicking `Close tab page` only closes the about page, leaving the new tab opened.

**Comment by bsclifton** [comment]
Is this the same issue as https://github.com/brave/browser-laptop/issues/9722?

**Comment by NejcZdovc** [comment]
@bsclifton no it's not the same.
@bridiver why did we added this (so that `about:newtab` is ignored) https://github.com/brave/browser-laptop/commit/f4283c0053569842773d5d6aca91825b55732ba3#diff-e12cc9b3579286a186d0f0c82dfa8186R66?

**Comment by bridiver** [comment]
@NejcZdovc it wasn't added, it was moved from https://github.com/brave/browser-laptop/commit/f4283c0053569842773d5d6aca91825b55732ba3#diff-f5a0b45b8323a469e78cf98fb0c49b8fL632

**Comment by NejcZdovc** [comment]
@bridiver you are correct. It was introduced here #6570 by me :smile: Not sure why I added it, not sure why and why it was working before.

**Comment by bsclifton** [comment]
cc @NejcZdovc for comment
Per conversation, he is saying this is happening ONLY for `about:newtab` because there is an if statement (or something to be discovered)

**Comment by bbondy** [comment]
0.19.x: 511bf20846d967d8cbfa57c5b681173ba5f5eeb3
0.20.x: 7bb52863e7412d225da791464849d2a35f101fd8
master: bf4726d9b7a780b133bb5144e08435ab798e0861
